src/serviceWorker.js

```diff
@@ -1,26 +1,3 @@
-/**
- * Prefetch a URL using the service worker.
- * @param {String} url The URL to prefetch
- */
-export async function prefetch(url) {
```

Comment:
I guess this is the breaking change?

Reply:
It would be, but I've just reverted that so that the prefetch function is still available on serviceWorker.js

Reply:
Nice